.github/workflows/publish_snapshot.yml

```diff
@@ -2,16 +2,16 @@ name: Publish Core Snapshot
 
```

Comment:
We need to change the name on line 1 from `Publish Core Snapshot` because it is not only core anymore.